src/main.rs

```diff
@@ -133,8 +134,10 @@ fn create_app<'a, 'b>(
         .arg(
             Arg::with_name("kernel")
                 .long("kernel")
-                .help("Path to loaded kernel. This may be a kernel or firmware that supports a PVH \
-                entry point, a vmlinux ELF file or a Linux bzImage or achitecture equivalent")
+                .help(
+                    "Path to loaded kernel. This may be a kernel or firmware that supports a PVH \
+                entry point, a vmlinux ELF file or a Linux bzImage or achitecture equivalent",
+                )
```

Comment:
nit: That's an unrelated formatting change

Reply:
I didn't change this, I think it was `cargo fmt`.

Reply:
That's weird because `cargo fmt` does not change anything on `master` but it does with my PR. So I think this is needed, otherwise the CI is going to complain that the code is not properly formatted.